src/main/java/application/Routes.java

```diff
@@ -28,10 +31,11 @@
 
         router.post("/echo_body", Handlers.EchoBody);
 
-        ROUTES = router.routes();
-    }
+        router.get("/redirect", Handlers.Redirect);
```

Comment:
This is neat. I don't think I've ever used a static block. Is the idea that you only ever want to run the router initialization once and have one version of it, like a singleton?

Reply:
Yeah, the thinking was that I wanted to lock down the route creation/access so that you couldn't inject new routes. Is there a way to do that that's more prevalent/idiomatic than a static block?

Reply:
Hm, you could still add routes to it if you wanted though right? Like in `main` I think I could do `Router router = Routes.ROUTER` and then `router.get(...)`. Doesn't really bother me though. What's the idea behind wanting to lock down the routes?

Reply:
Ah shoot I think you're right -- the idea was that say the router has a route for `/simple_get` with just a `HEAD` method associated with it, so hitting it with any other method should return a method not allowed response, but if you can just define another method for that path you could get around that.